.toolversions

```diff
@@ -1,3 +1,4 @@
 Microsoft.DotNet.BuildTools=1.0.27-prerelease-01205-03
 Microsoft.DotNet.BuildTools.Run=1.0.1-prerelease-01205-03
+Microsoft.DotNet.Build.Tasks.Feed=1.0.0-prerelease-02119-02
```

Comment:
Is this actually being used in BuildTools?

Reply:
Not at the moment, this would be used for publishing to the transport feed during official builds.  The alternative I considered was to just use the built packages from the build, but I figured that would lead to a chicken and egg problem so I'd rather use a stable method.

Reply:
I'm fine with it being used by I don't see you plugging it into the publish step. Am I missing that somewhere?

Reply:
The publishing happens in the VSTS definition (which is not checked in for BuildTools), so that's a subsequent (hidden) update, unfortunately.